### ISSUE TITLE
Accessibility Labels Missing for BCSC Personal Credential Steps Screen

### ISSUE THREAD

**Issue by travkhl** [bcgov/bc-wallet-mobile#2564]
Accessibility labels are inconsistently disabled for steps.

![Image](https://github.com/user-attachments/assets/042a0dd9-e281-4183-9303-c492df0f4cf9)
